playbooks/tags.json

```diff
@@ -1,3 +1,6 @@
 [
+    "FSRSolutionPack",
+    "impossibleTravellerThreatResponse-1.0.0",
```

Comment:
is it expected

Reply:
hi @cs-suyog-jinturkar I have removed the unnecessary tags from the playbook